lib/features/email/presentation/controller/single_email_controller.dart

```diff
@@ -414,8 +422,8 @@ class SingleEmailController extends BaseController with AppLoaderMixin {
   void _resetToOriginalValue() {
     attachmentsExpandMode.value = ExpandMode.COLLAPSE;
     emailAddressExpandMode.value = ExpandMode.COLLAPSE;
-    emailContents.clear();
-    initialEmailContents?.clear();
+    emailContents.value = null;
+    initialEmailContents = null;
```

Comment:
what happen? How about clear?

Reply:
IMO: 
I change from `List<EmailContents> ` to `String`
With the old logic we will clear these values when handle opens the email view. So I will set it to the value lun